src/backend/metal/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "gfx-backend-metal"
-version = "0.2.2"
+version = "0.2.3"
```

Comment:
there is no need for this bump: we aren't publishing the patch versions from `master` anyway

Reply:
Reverted that and squashed. I was hoping to get a published metal backend that builds for iOS sim.

Reply:
sure, let's back-port this to hal-0.2 branch with the patch version bump